services/src/main/java/org/keycloak/services/resources/admin/RealmAdminResource.java

```diff
@@ -416,10 +420,25 @@ public void updateRealmEventsConfig(final RealmEventsConfigRepresentation rep) {
         }
         
         if(dateFrom != null) {
-            query.fromDate(dateFrom);
+            SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd");
+            Date from = null;
+            try {
+                from = df.parse(dateFrom);
+            } catch (ParseException e) {
+                return new ArrayList<Event>();
```

Comment:
This should return an error or throw an exception - not just return an empty list of events